services/QuillLMS/app/workers/delete_obsolete_active_activity_sessions_worker.rb

```diff
@@ -4,13 +4,17 @@ class DeleteObsoleteActiveActivitySessionsWorker
   include Sidekiq::Worker
   sidekiq_options queue: SidekiqQueue::LOW
 
+  BATCH_SIZE = 100
+
   def perform
```

Comment:
Note that `delete_all` ["Returns the number of rows affected."](https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-delete_all) as an integer.